webapi/src/main/resources/knora-ontologies/knora-base.ttl

```diff
@@ -2327,7 +2405,22 @@
                          owl:cardinality    "1"^^xsd:nonNegativeInteger ] ,
                        [ rdf:type           owl:Restriction ;
                          owl:onProperty     :hasSegmentBounds ;
-                         owl:cardinality    "1"^^xsd:nonNegativeInteger ] ;
+                         owl:cardinality    "1"^^xsd:nonNegativeInteger ] ,
+                       [ rdf:type           owl:Restriction ;
+                         owl:onProperty     :hasTitle ;
+                         owl:maxCardinality "1"^^xsd:nonNegativeInteger ] ,
+                       [ rdf:type           owl:Restriction ;
+                         owl:onProperty     :hasDescription ;
+                         owl:minCardinality "0"^^xsd:nonNegativeInteger ] ,
```

Comment:
**question**: A `Segment` may have more than one description?

I would have expected a single description to be optional.

Reply:
That was my first assumption too, but turns out that projects may want to be able to describe different aspects of a Segment as different descriptions. I'd have been happy either way, but this aligns with how EKWS modelled their Sequences, which I found reason enough to go that way.